### PR TITLE
Adding Temperature Probes

### PR DESCRIPTION
**Pull request by TaylorSmith28**
Implementing the Temperature Probes

### CODE REVIEW

File: titration/ui_state/demo_mode/demo_temp_probe.py [hunk 1]
```diff
@@ -54,37 +46,47 @@ def loop(self):
         The function to loop through and display to the LCD screen until a new keypad input
         """
         if self.substate == 1:
-            self.titrator.lcd.print("1: Get Temperature", line=1)
```
Comment (prestoncarman) [inline titration/ui_state/demo_mode/demo_temp_probe.py:57]: What about get probe 1 and 2? Then on the probe screen show both temp and resistance.

File: titration/ui_state/demo_mode/demo_temp_probe.py [hunk 2]
```diff
@@ -54,37 +46,47 @@ def loop(self):
...
 
-            self.titrator.temp_controller.update()
+        elif self.substate == 5:
+            self.titrator.lcd.print("Probe Two Res", line=1)
+            self.titrator.lcd.print(
+                f"{self.titrator.temp_probe_two.get_resistance()} Ohms",
+                line=2,
+                style="center",
+            )
+            self.titrator.lcd.print("Press any to cont.", line=3)
```
Comment (prestoncarman) [inline titration/ui_state/demo_mode/demo_temp_probe.py:91]: Does this text fit? "Any key to continue" This way you don't need an abbreviation.

**Review by prestoncarman — CHANGES_REQUESTED** [review]
This change improves things. I think it would be better to have two menu options: probe one and probe two. Then when you go to a probe you see all the information reported from the probe: temperature and resistance. Simple the view and number of clicks to view all the information.

File: titration/ui_state/demo_mode/demo_temp_probe.py (hunk 1 above)
Comment (TaylorSmith28) [reply titration/ui_state/demo_mode/demo_temp_probe.py:57]: Good idea! I will implement this.

File: titration/ui_state/demo_mode/demo_temp_probe.py (hunk 2 above)
Comment (TaylorSmith28) [reply titration/ui_state/demo_mode/demo_temp_probe.py:91]: There are many UI_States that use the phrase "Press any to cont.", would you like me to just create a separate UI PR that addresses all the UI States that uses this wording?

**Review by prestoncarman — APPROVED** [review]

**Pull request merged**